cluster/gce/util.sh

```diff
@@ -156,8 +156,40 @@ function copy-if-not-staged() {
   fi
 }
 
+# Prepare a tarball of kube-system manifests for trusty based cluster.
+#
+# Vars set:
+#   KUBE_MANIFESTS_TAR_URL
+#   KUBE_MANIFESTS_TAR_HASH
+function prepare-manifests-tar() {
```

Comment:
I have not figured what to change for official releases. So I put a TODO above the place where this function is called to remind me for adding the release support. I am working on it and will make it happen in the next PR (coming next week). I separated this PR out so as to unblock other master side changes. I think this PR and my next PR for release support would be appear in the same release, so it should be fine.